CMakeLists.txt

```diff
@@ -580,7 +598,13 @@ add_definitions(-DUNICODE)
 add_definitions(-D_UNICODE)
 
 if (IS_64BIT_BUILD EQUAL 1)
-  set(ARCH_SOURCES_DIR amd64)
+  if (CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64 EQUAL 1)
+    set(ARCH_SOURCES_DIR amd64)
+  elseif (CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM64 EQUAL 1)
+    set(ARCH_SOURCES_DIR arm64)
+  else ()
+    message(FATAL_ERROR "Only amd64 and arm64 are supported for 64-bit builds")
```

Comment:
Misses Windows too (is causing the build failure), same elseif(WIN32) would work here at least temporarily.